tests/TestCase/Mailer/MailerTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+namespace Cake\Test\TestCase\Mailer;
+
+use Cake\Mailer\Email;
+use Cake\Mailer\Mailer;
+use Cake\TestSuite\TestCase;
+
+class TestMailer extends Mailer
+{
+    public function getEmailForAssertion()
```

Comment:
I am not a fan of giving access to unneeded stuff but we could. Aside from tests, I don't see why one would need it using the mailer. The goal was to isolate mail related stuff for easier testing, etc.